### ISSUE TITLE
Set Cookie broken in last update

### ISSUE THREAD

**Issue by ldionmarcil** [synfron/ReshaperForBurp#58]
Hey, since the release "Reshaper for Burp v2.3.2", the "Set Value: Request Cookie" has been behaving incorrectly. Prior to that version, Reshaper would add value to a cookie, as expected. Since this release however, Reshaper will add a header with the value of the cookie name. I don't understand this behaviour, I think it's a bug.
Example:
<img width="1067" alt="image" src="https://github.com/synfron/ReshaperForBurp/assets/1320868/54c856ae-ae94-4133-aeb8-8793b4538254">

I expect this configuration to create/modify the Cookie header with the value `Foobar=Barbaz;`. However, this is what is added to my requests:
`Foobar: Foobar=Barbaz`.

This worked properly prior to this release, but I noticed your last commits modified the cookie handling logic. Is this intended? If so, how do we add cookies to requests? Thank you. 

Here is a configuration for you to test this:
```yaml
rules:
- whens:
  - !<.WhenFromTool>
    negate: false
    useOrCondition: false
    tool: "Proxy"
  thens:
  - !<.ThenSetValue>
    useMessageValue: false
    sourceMessageValue: "SourceAddress"
    sourceIdentifier: ""
    sourceIdentifierPlacement: "Last"
    sourceMessageValueType: "Text"
    sourceMessageValuePath: ""
    useReplace: false
    regexPattern: ""
    text: "Barbaz"
    replacementText: ""
    destinationMessageValueType: "Text"
    destinationMessageValuePath: ""
    destinationMessageValue: "HttpRequestCookie"
    destinationIdentifier: "Foobar"
    destinationIdentifierPlacement: "New"
  enabled: true
  autoRun: true
  name: "bug repro 2"
webSocketRules: []
variables: []

```

**Comment by ddwightx** [comment]
Sorry for the inconvenience. I'll take a look. Let me know if you need an alternative way to set this in the mean time while I look at this issue.

**Comment by ddwightx** [comment]
This issue will be fixed in v2.3.3. See https://github.com/synfron/ReshaperForBurp/releases/tag/v2.3.3
